provider/equinix/environ.go

```diff
@@ -17,6 +19,7 @@ import (
 	"github.com/juju/loggo"
 	"github.com/juju/retry"
 	"github.com/juju/schema"
+	"github.com/juju/utils/ssh"
```

Comment:
Should be `github.com/juju/utils/v3/ssh`

Reply:
https://github.com/juju/juju/compare/9ce32f0f7b0236793d0771b8cdae0dfbd6207b6f..3a6cdc2210c23104b1b3cf867dce9907035060e3